homework/css/index.css

```diff
@@ -0,0 +1,51 @@
+body {
```

Comment:
As I mentioned, avoid adding layout styles to body. Better add a container with appropriate class. And style it. Body should get only very general styling

Reply:
Done